packages/w3/bin.js

```diff
@@ -11,6 +11,17 @@ cli
   .example('put path/to/files')
   .example('get bafkreigh2akiscaildcqabsyg3dfr6chu3fgpregiymsck7e7aqa4s52zy -o room-guardian.jpg')
 
+cli.command('token')
+  .option('--api', 'URL for API to use')
+  .option('--token', 'API token to use')
```

Comment:
should we drop this and only allow token via prompt to not be pasted into commands history?

Reply:
it's your command history. if you wanna paste in a token, go nuttsss i say!

Reply:
yes, my point here is mostly that this will appear in the help of the command, which will guide people towards providing the token as a CLI parameter, as they will likely not understand they will be prompted